### DISCUSSION TITLE
Logical AND in Notification

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Planner#1421] [Feature Requests]
For example, if you set notification for category A and B, our notification will fire a notification when the post either has category A or B.

It will be great if it only fire when both condition applied.

Original request: https://wordpress.org/support/topic/using-a-logical-and-in-taxonomy/

**Comment by whydontdoit** [comment]
I agree ...and what about adding option condition for applying notification only on specific post statuses, not only on changing status event?
